README.md

```diff
@@ -0,0 +1,372 @@
+
+```
+$ moactl create idp --cluster rh-moa-test-cluster1 --type github                                                                                               
+I: Loading cluster 'rh-moa-test-cluster1'                                                                                                                                                               
+I: Loading identity providers for cluster 'rh-moa-test-cluster1'                                                                                                                                        
+To use GitHub as an identity provider, you must first register the application:                                                                                                                 
+? List of GitHub organizations or teams that will have access to this cluster: openshift-online                                                                                                 
+* Open the following URL: https://github.com/organizations/openshift-online/settings/applications/new?oauth_application%5Bcallback_url%5D=https%3A%2F%2Foauth-openshift.apps.rh-moa-test-cluster1.j9n4.s
+1.devshift.org%2Foauth2callback%2Fgithub-1&oauth_application%5Bname%5D=rh-moa-test-cluster1&oauth_application%5Burl%5D=https%3A%2F%2Fconsole-openshift-console.apps.rh-moa-test-cluster1.j9n4.s1.devshift.org   
+* Click on 'Register application'                                                                                                                                                               
+? Copy the Client ID provided by GitHub: &lt;my-github-client-id&gt;                                                                                                                                   
+? Copy the Client Secret provided by GitHub: &lt;my-github-client-secret&gt;                                                                                                           
+I: Configuring IDP for cluster 'rh-moa-test-cluster1'                                                                                                                                                   
+I: Identity Provider 'github-1' has been created. You need to ensure that there is a list of cluster administrators defined. See `moactl user add --help` for more information. To login into th
+e console, open https://console-openshift-console.apps.rh-moa-test-cluster1.j9n4.s1.devshift.org and click on github-1
+```
+
+The IDP can take 1-2 minutes to be configured within your cluster.
+
+Run the following command to verify that your IDP has been configured correctly:
+
+```
+$ moactl list idps --cluster rh-moa-test-cluster1
+NAME        TYPE      AUTH URL
+github-1    GitHub    https://oauth-openshift.apps.rh-moa-test-cluster1.j9n4.s1.devshift.org/oauth2callback/github-1
+```
```

Comment:
Here let's have the user try to login to the console with their github credentials before moving on to creating a dedicated-admin.

Reply:
And maybe a screenshot or 2 will help.

Reply:
I moved the content but leaving this thread open so I remember to add some screenshots.